src/gradualizer_db.erl

```diff
@@ -189,8 +189,8 @@ handle_call({load, Filename}, _From, State) ->
                                        types  = maps:merge(Ty1, Ty2),
                                        loaded = maps:merge(Loaded1, Loaded2)},
                 {reply, ok, NewState}
-            catch error:E ->
-                {reply, {error, E, erlang:get_stacktrace()}, State}
+            catch _:Exception:Stacktrace ->
```

Comment:
(my opinion) please don't use the new syntax introduced only in Erlang/OTP 21.0. It would be good to keep supporting older Erlang versions (current master still runs fine on OTP 19+). The old syntax is still valid (only results in a warning). 

Let's use this directive proposed by @zuiderkwast (in https://github.com/josefs/Gradualizer/pull/18) in the top of the module to silence this warning for the time being.
```
-ifdef(OTP_RELEASE).
-compile([{nowarn_deprecated_function,{erlang,get_stacktrace,0}}]).
-endif.
```

If we start to need stacktraces in many places we can start using https://hex.pm/packages/stacktrace_compat, but for now I wanted to avoid any external dependency.

(this PR is great work btw 👍)

Reply:
Thanks, let's use it!
There are 2 other compiler warnings:
```
src/ty.erl:6: Warning: export_all flag enabled - all functions will be exported
src/typechecker.erl:9: Warning: export_all flag enabled - all functions will be exported
```
I think it's nice to make proper `export` here